src/python/pants/option/options_bootstrapper_test.py

```diff
@@ -39,9 +39,9 @@ def assert_bootstrap_options(
                     fp.write(f"{k} = {repr(v)}\n")
             fp.close()
 
-            args = [*self._config_path(fp.name), *(args or [])]
+            args = ["pants", *self._config_path(fp.name), *(args or [])]
```

Comment:
The OptionsBootstrapper now uniformly requires that the args start with `argv[0]`, so...